src/main/java/com/denizenscript/clientizen/events/ScreenOpenCloseEvent.java

```diff
@@ -17,49 +27,163 @@ public class ScreenOpenCloseEvent extends ScriptEvent {
+        registerScreenName(EnchantmentScreen.class, "enchantment");
+        registerScreenName(FurnaceScreen.class, "furnace");
+        registerScreenName(Generic3x3ContainerScreen.class, "generic_3x3_container");
+        registerScreenName(GenericContainerScreen.class, "generic_container");
+        registerScreenName(GrindstoneScreen.class, "grindstone");
+        registerScreenName(HangingSignEditScreen.class, "hanging_sign_edit");
+        registerScreenName(HopperScreen.class, "hopper");
+        registerScreenName(HopperScreen.class, "horse");
+        registerScreenName(InventoryScreen.class, "inventory");
+        registerScreenName(JigsawBlockScreen.class, "jigsaw");
+        registerScreenName(LecternScreen.class, "lectern");
+        registerScreenName(LoomScreen.class, "loom");
+        registerScreenName(MerchantScreen.class, "merchant");
+        registerScreenName(MinecartCommandBlockScreen.class, "command_block_minecart");
+        registerScreenName(ShulkerBoxScreen.class, "shulker_box");
+        registerScreenName(SignEditScreen.class, "sign_edit");
+        registerScreenName(SmithingScreen.class, "smithing");
+        registerScreenName(SmokerScreen.class, "smoker");
+        registerScreenName(StonecutterScreen.class, "stunecutter");
+        registerScreenName(StructureBlockScreen.class, "structure_block");
+        // Multiplayer screens
+        registerScreenName(MultiplayerScreen.class, "multiplayer");
+        registerScreenName(MultiplayerWarningScreen.class, "multiplayer_warning");
+        registerScreenName(SocialInteractionsScreen.class, "social_interactions");
+        // Option screens
+        registerScreenName(AccessibilityOptionsScreen.class, "accessibility_options");
+        registerScreenName(ChatOptionsScreen.class, "chat_options");
+        registerScreenName(ControlsOptionsScreen.class, "controls_options");
+        registerScreenName(CreditsAndAttributionScreen.class, "credits_and_attribution");
+        registerScreenName(GameOptionsScreen.class, "game_options");
+        registerScreenName(KeybindsScreen.class, "keybinds");
+        registerScreenName(LanguageOptionsScreen.class, "language_options");
+        registerScreenName(MouseOptionsScreen.class, "mouse_options");
+        registerScreenName(OnlineOptionsScreen.class, "online_options");
+        registerScreenName(OptionsScreen.class, "options");
+        registerScreenName(SkinOptionsScreen.class, "skin_options");
+        registerScreenName(SoundOptionsScreen.class, "sound_options");
+        registerScreenName(TelemetryInfoScreen.class, "telemetry_info");
+        registerScreenName(VideoOptionsScreen.class, "video_options");
+        // Packs screens
+        registerScreenName(ExperimentalWarningScreen.class, "experimental_warning");
+        registerScreenName((Class<? extends Screen>) ExperimentalWarningScreen.class.getDeclaredClasses()[0], "experimental_warning_details");
```

Comment:
Use an access widener instead of reflection

Reply:
Added one